### ISSUE TITLE
Xbrz 1.5 released

### ISSUE THREAD

**Issue by LuismaSP89** [gonetz/GLideN64#1644]
Xbrz 1.5 has been released. This is not an issue, but i think Gliden64 can be faster with this plugin with the latest improvements.

**Comment by retrobenny** [comment]
Honestly haven't seen the slightest decrease in performance when using Xbrz as an enhancement,even on less powerful hardware.
Maybe its GPU related if any performance impact is involved.
Wouldn't mind seeing it done if there can potentially be an end-result like fixing the broken text issues in games like Banjo-Tooie at least seen on Android.

**Comment by LuismaSP89** [comment]
@retrobenny  I see permormance drops in some games (With 6xbrz) specially in goldeneye. 

I think that yes, the performance of the xbrz filter it´s GPU related. Btw i got a i7 4790k and a GTX 970, and even with that, all updates of performance with the xbrz filter are welcome.

**Comment by gonetz** [comment]
GLideNHQ applies texture filters on CPU side. GPU works with already filtered textures.   Thus, even most complex filters like xbrz work with no problems on any GPU.

**Comment by gonetz** [comment]
@LuismaSP89  
>Xbrz 1.5 has been released. 

Here should be link to c/c++ sources.

**Comment by Eignar17** [comment]
https://sourceforge.net/projects/xbrz/files/ScummVM/

**Comment by gonetz** [comment]
@Eignar17 it is link to ScummVM sources. These sources do not contain xbrz sources. Sources to xBRZ are here: https://sourceforge.net/projects/xbrz/files/xBRZ/ 
but there is only 1.4

**Comment by retrobenny** [comment]
Sad thing is that 1.5 has supposedly been around since August,but where would one acquire the source?

**Comment by Eignar17** [comment]
@gonetz contact for zenju creator of xbrz

**Comment by gonetz** [comment]
No, thanks. I can wait until 1.5 will be officially published somewhere.

**Comment by Eignar17** [comment]
Does this serve you? https://sourceforge.net/projects/xbrz/files/Kega%20Fusion/Kega-Fusion364%2BxBRZ_Source.7z/download

**Comment by LuismaSP89** [comment]
I´ll try to check it out daily and report any update of the code.

Btw i send a email to Zenju for another question, if he anwers, i´ll ask him for the 1.5 code.

**Comment by LuismaSP89** [comment]
@gonetz  Finally, Zenju has released officially xbrz 1.5 : )

https://sourceforge.net/projects/xbrz/files/xBRZ/

**Comment by gonetz** [comment]
```
===========
|Changelog|
===========

xBRZ 1.5 [2017-08-07]
---------------------
Added RGB conversion routines

```

Is it worth to update? I already implemented RGB conversion in 1.4.

**Comment by LuismaSP89** [comment]
@gonetz I don't know if the improvements of this version worths and update, i only advice that there's an update xD. Sorry, i have no programming skills.

**Comment by rez-** [comment]
xBRZ 1.6 [2018-02-27]
---------------------
Added bilinear scaling
Updated license info
Option to skip color buffer creation
